build.rs

```diff
@@ -342,6 +341,8 @@ fn static_linking(env_vars: &EnvVars) {
                 .expect("Cannot write binding to file.");
         }
```

Comment:
Please add a return here, then the cfg on the panic can be removed.

Reply:
> the cfg on the panic can be removed

Then you'll get an `unreachable statement` warning as well.

Reply:
fine